sbncode/CAFMaker/FillReco.cxx

```diff
@@ -733,6 +735,9 @@ namespace caf
       CopyPropertyIfSet(propertiesMap, "LArThreeDPCAFeatureTool_TertiaryPCARatio",           srpfp.pfochar.pca3ratio);
       CopyPropertyIfSet(propertiesMap, "LArThreeDVertexDistanceFeatureTool_VertexDistance",  srpfp.pfochar.vtxdist);
     }
+    if (t0) {
```

Comment:
Do we want to set some value for if !t0 case? If I'm not mistaken it will otherwise be nan in the CAF? Maybe I'm missing where some default is set.